examples/select.html

```diff
@@ -38,6 +38,7 @@
             L.latLng(51.50,-0.10),
           ],
           mode: 'lock',
+             alt:'img',
```

Comment:
Hi, is this for accessibility, or is it otherwise related to ordering? Or just so you can see the effects in the console? Maybe after we confirm it's working in various scenarios, we can remove these so that the examples stay as simple as possible?

Reply:
Alright, sounds good
It was for the ordering 
will take it out @jywarren